src/configs/rlang.rs

```diff
@@ -0,0 +1,29 @@
+use crate::config::ModuleConfig;
+
+use serde::Serialize;
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig, Serialize)]
+pub struct RLangConfig<'a> {
+    pub format: &'a str,
+    pub style: &'a str,
+    pub symbol: &'a str,
+    pub disabled: bool,
+    pub detect_extensions: Vec<&'a str>,
+    pub detect_files: Vec<&'a str>,
+    pub detect_folders: Vec<&'a str>,
+}
+
+impl<'a> Default for RLangConfig<'a> {
+    fn default() -> Self {
+        RLangConfig {
+            format: "via [$symbol($version )]($style)",
+            style: "blue bold",
+            symbol: "📐 ",
+            disabled: false,
+            detect_extensions: vec!["R", "Rd", "Rmd", "Rproj", "Rsx"],
```

Comment:
I think you should make these lowercase. I believe it should work for any case?

Reply:
^^ same with all others

Reply:
Can you tell me your thinking for it to be lowercase? is that a convention that starship has?

For correctness and if possible, I would keep it as-is, unless another reason supplants it. The `R` file extensions are normally upper case for R source code (not enforced but the standard), either from its own [source code](https://svn.r-project.org/R/trunk/src/library/base/R/), [its libraries](https://github.com/tidyverse/tidyverse) or by files generated by tools such as RStudio (the official IDE).

Reply:
That's just how I do it and most other modules do it like that so for consistency I suppose

Reply:
I think correctness is more important than consistency here.

Reply:
I think correctness is more important than consistency here.